Appraisals

```diff
@@ -28,6 +28,12 @@ appraise 'rails-7-1' do
   gem 'activerecord', '~> 7.1.0'
 end
 
+appraise 'rails-7-2' do
+  gem 'actionmailer', '~> 7.2.0'
+  gem 'activejob', '~> 7.2.0'
+  gem 'activerecord', '~> 7.2.0'
+end
```

Comment:
TODO: Finish Appraisal set up for Rails 7.2

Reply:
Do you intend to do that in this PR?

(While you're at it, mind bumping the `delayed` gem's version?)

Reply:
Yep. Was struggling to get the incantation working to install mysql. Should be good now

Reply:
I think the version bump requires an update to all of the lockfiles.

Reply:
Bumped